src/Microsoft.ApiDesignGuidelines.Analyzers/Microsoft.ApiDesignGuidelines.Analyzers.md

```diff
@@ -211,6 +339,44 @@ Severity: Warning
 
 Help: [https://msdn.microsoft.com/en-us/library/ms182253.aspx](https://msdn.microsoft.com/en-us/library/ms182253.aspx)
 
+### CA1724: Type names should not match namespaces ###
+
+Type names should not match the names of namespaces that are defined in the .NET Framework class library. Violating this rule can reduce the usability of the library.
+
```

Comment:
The analyzer implemented by @basoundr also warns of namespace names matching types in your own project. I assume that's how the FxCop rule worked, right Balaji? (Remember that our initial goal is maximum possible compatibility with FxCop.) If so, mention that in the description as well.